app/assets/stylesheets/all/components/_rtl.css.scss

```diff
@@ -1,9 +1,34 @@
 html[dir="rtl"] {
-  #userinfo {
-    float: left;
 
-    .icon-btn {
+  #content {
+    h1.title{
       float: right;
+      margin-right: 0;
```

Comment:
So this is the kind of stuff I'd like to see us use:

http://matanich.com/2013/09/06/rtl-css-with-sass

I'd rather not put all of these separate styles for RTL in a separate file. It will be difficult to remember to keep it up to date.